test/metabase/query_processor_test/explicit_joins_test.clj

```diff
@@ -269,10 +269,20 @@
                 [3 "Kaneonuskatew Eiran" "2014-11-06T16:15:00Z" 3 "2014-09-15T00:00:00Z" 8 56]]
                rows))))))
 
+(defmethod driver/database-supports? [::driver/driver ::foreign-keys-as-required-by-tests]
+  [_driver _feature _database]
+  true)
+
+;;; mongodb doesn't support foreign keys required by some tests below.
+;;;
+;;; TODO -- not sure what exactly this means. Maybe it was talking about marking FKs automatically during sync? Since we
+;;; now do that manually for DBs like MongoDB maybe we can enable these tests for Mongo.
+(defmethod driver/database-supports? [:mongo ::foreign-keys-as-required-by-tests]
+  [_driver _feature _database]
+  false)
+
 (deftest ^:parallel select-*-source-query-test
-  (mt/test-drivers (disj (mt/normal-drivers-with-feature :left-join)
-                         ;; mongodb doesn't support foreign keys required by this test
-                         :mongo)
+  (mt/test-drivers (mt/normal-drivers-with-feature :left-join ::foreign-keys-as-required-by-tests)
```

Comment:
Honestly, no idea what this means or if it's still relevant. We have a lot of tests with implicit joins we're skipping against MongoDB, we might actually be able to enable them now since we're marking FK relationships when loading test data. I didn't want to go down that rabbit hole in this PR however.

Reply:
I think there is one more pass required (after my recent PR), searching for `;; mongodb doesn't support foreign keys required by this test` and verifying whether test could be enabled. Opened task https://github.com/metabase/metabase/issues/46836 to track that.